src/Item.php

```diff
@@ -2241,8 +2241,8 @@ public function get_enclosures()
                 }
             }
 
-            if ($enclosure = $this->get_item_tags(\SimplePie\SimplePie::NAMESPACE_RSS_20, 'enclosure')) {
-                if (isset($enclosure[0]['attribs']['']['url'])) {
+            foreach ((array) $this->get_item_tags(\SimplePie\SimplePie::NAMESPACE_RSS_20, 'enclosure') as $enclosure) {
```

Comment:
I assume the cast is for the [`null` case](https://github.com/simplepie/simplepie/blob/839d34c10273f7eb2abe52fbb78a09d5727e14f9/src/Item.php#L152)?

If that is the case, something like the following might be less confusing.

```suggestion
            foreach ($this->get_item_tags(\SimplePie\SimplePie::NAMESPACE_RSS_20, 'enclosure') ?? [] as $enclosure) {
```

But maybe even better would be changing the `get_item_tags` to return empty array so that it actually respects the `@return` type annotation.

Reply:
Yes. I had kept the exact same syntax than lines above